src/lib/GovernorTimelockCompound.sol

```diff
@@ -0,0 +1,246 @@
+    bytes[] memory calldatas,
+    bytes32 /*descriptionHash*/
+  ) internal virtual override {
+    uint256 eta = proposalEta(proposalId);
+    require(eta > 0, "GovernorTimelockCompound: proposal not yet queued");
+    // In the original contract, the _timelock is not casted to an address. Failing to cast it causes a compile error "Explicit type conversion not allowed from "contract IRadworksTimelock" to "address payable".".
+    // We explicitly cast to an address to solve this error.
+    //
+    // Original Openzeppelin line:
+    // https://github.com/OpenZeppelin/openzeppelin-contracts/blob/49c0e4370d0cc50ea6090709e3835a3091e33ee2/contracts/governance/extensions/GovernorTimelockCompound.sol#L128
+    Address.sendValue(payable(address(_timelock)), msg.value);
+    for (uint256 i = 0; i < targets.length; ++i) {
+      _timelock.executeTransaction(targets[i], values[i], "", calldatas[i], eta);
+    }
+  }
+
+  /**
+   * @dev Overridden version of the {Governor-_cancel} function to cancel the timelocked proposal if it as already 
+   * been queued.
+   */
+  function _cancel(
+    address[] memory targets,
+    uint256[] memory values,
+    bytes[] memory calldatas,
+    bytes32 descriptionHash
+  ) internal virtual override returns (uint256) {
+    uint256 proposalId = super._cancel(targets, values, calldatas, descriptionHash);
+
+    uint256 eta = proposalEta(proposalId);
+    if (eta > 0) {
+      for (uint256 i = 0; i < targets.length; ++i) {
+        _timelock.cancelTransaction(targets[i], values[i], "", calldatas[i], eta);
+      }
+      _proposalTimelocks[proposalId].timer.reset();
+    }
+
+    return proposalId;
+  }
+
+  /**
+   * @dev Address through which the governor executes action. In this case, the timelock.
+   */
+  function _executor() internal view virtual override returns (address) {
+    return address(_timelock);
+  }
+
+  /**
+   * @dev Accept admin right over the timelock.
+   */
+  // solhint-disable-next-line private-vars-leading-underscore
+  function __acceptAdmin() public {
+    _timelock.acceptAdmin();
+  }
+
+  /**
+   * @dev Public endpoint to update the underlying timelock instance. Restricted to the timelock itself, so updates
+   * must be proposed, scheduled, and executed through governance proposals.
+   *
+   * For security reasons, the timelock must be handed over to another admin before setting up a new one. The two
+   * operations (hand over the timelock) and do the update can be batched in a single proposal.
+   *
+   * Note that if the timelock admin has been handed over in a previous operation, we refuse updates made through the
+   * timelock if admin of the timelock has already been accepted and the operation is executed outside the scope of
+   * governance.
+
+   * CAUTION: It is not recommended to change the timelock while there are other queued governance proposals.
+   * @dev The interface for `newTimelock` was changed to conform to the Radworks Timelock interface.
```

Comment:
if you're going beyond general "this was modified" comments, consider something more specific

```suggestion
   * @dev `ICompoundTimelock newTimelock` type was changed to `IRadworksTimelock`.
```

Reply:
Went with shorter "this was modified" 4c8b4aa5079ff2ca8a0124eeab3624cc5cfce0db